EIPS/eip-6969.md

```diff
@@ -0,0 +1,76 @@
+---
+eip: 6969
+title: Submit Block RPC
+author: Zachary Belford (@belfordz)
+status: Draft
+type: Standards Track
+category: Interface
+created: 2019-06-26
+---
```

Comment:
This should depend on 1474.

Reply:
Semi-related... have you seen OpenRPC or https://github.com/etclabscore/ethereum-json-rpc-specification ?

Reply:
I've taken the liberty to update 1474 to something more maintainable.

https://github.com/BelfordZ/EIPs/blob/feat/json-rpc-spec/EIPS/eip-3333.md